### DISCUSSION TITLE
Vector3 swizzle

### DISCUSSION THREAD

**Discussion by nenad-jalsovec** [godotengine/godot-proposals#9807] [Engine Core]
How about adding swizzle to Vector3 class? It could be in the form of methods like `some_vector.xxy()`, or better yet similar to glsl swizzle simply `some_vector.xxy`.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/727
